site/content/contribute/server/developer-setup/arch.md

```diff
@@ -75,3 +75,9 @@
     ```
 
     **Note:** Browsing directly to http://localhost:8065/ will display a `404 Not Found` until the web app is configured. See [Web App Developer Setup](https://developers.mattermost.com/contribute/webapp/developer-setup/) and [Mobile App Developer Setup](https://developers.mattermost.com/contribute/mobile/developer-setup/) for additional setup.
+
+    The `stop-server` make target does not stop all the docker containers started by `run-server`. To stop the running docker containers:
+
+    ```sh
+    make stop-docker
```

Comment:
I did a PR that add the to the `make stop` the stop-docker step, i feel this was missing: https://github.com/mattermost/mattermost-server/pull/13020